pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -192,7 +199,6 @@ stdenv.mkDerivation (finalAttrs: {
     ./0016-inherit-systemd-environment-when-calling-generators.patch
     ./0017-core-don-t-taint-on-unmerged-usr.patch
     ./0018-tpm2_context_init-fix-driver-name-checking.patch
-    ./0019-bootctl-also-print-efi-files-not-owned-by-systemd-in.patch
   ] ++ lib.optional stdenv.hostPlatform.isMusl (
```

Comment:
[Updated musl patches are available.](https://git.openembedded.org/openembedded-core/tree/meta/recipes-core/systemd/systemd_254.bb?id=69d37620a7a0626eaee1c0b37057be2ac2687290#n35)  Up to you whether you want to include them here or leave it for somebody else to fix during staging as follow up.

Reply:
Never touched the musl stuff, so I'd leave it to someone else.

Reply:
I will pull them.

Reply:
I tried: `pkgsMusl.systemdMinimal` compiles, `pkgsMusl.systemd` does not, here's the diff:

```diff
diff --git a/pkgs/os-specific/linux/systemd/default.nix b/pkgs/os-specific/linux/systemd/default.nix
index e3e95246a0bf..d14ac5880f63 100644
--- a/pkgs/os-specific/linux/systemd/default.nix
+++ b/pkgs/os-specific/linux/systemd/default.nix
@@ -204,16 +204,16 @@ stdenv.mkDerivation (finalAttrs: {
   ] ++ lib.optional stdenv.hostPlatform.isMusl (
     let
       oe-core = fetchzip {
-        url = "https://git.openembedded.org/openembedded-core/snapshot/openembedded-core-f34f6ab04b443608497b73668365819343d0c2fe.tar.gz";
-        sha256 = "DFcLPvjQIxGEDADpP232ZRd7cOEKt6B48Ah29nIGTt4=";
+        url = "https://git.openembedded.org/openembedded-core/snapshot/openembedded-core-69d37620a7a0626eaee1c0b37057be2ac2687290.tar.gz";
+        hash = "sha256-oq7hTDdcimBxyq3qPAbqv5GP8rkygkyda63SFNXxXmk=";
       };
       musl-patches = oe-core + "/meta/recipes-core/systemd/systemd";
     in
     [
       (musl-patches + "/0001-Adjust-for-musl-headers.patch")
+      (musl-patches + "/0003-errno-util-Make-STRERROR-portable-for-musl.patch")
       (musl-patches + "/0005-pass-correct-parameters-to-getdents64.patch")
       (musl-patches + "/0006-test-bus-error-strerror-is-assumed-to-be-GNU-specifi.patch")
-      (musl-patches + "/0007-Add-sys-stat.h-for-S_IFDIR.patch")
       (musl-patches + "/0009-missing_type.h-add-comparison_fn_t.patch")
       (musl-patches + "/0010-add-fallback-parse_printf_format-implementation.patch")
       (musl-patches + "/0011-src-basic-missing.h-check-for-missing-strndupa.patch")
@@ -229,7 +229,11 @@ stdenv.mkDerivation (finalAttrs: {
       (musl-patches + "/0022-Handle-__cpu_mask-usage.patch")
       (musl-patches + "/0023-Handle-missing-gshadow.patch")
       (musl-patches + "/0024-missing_syscall.h-Define-MIPS-ABI-defines-for-musl.patch")
-      (musl-patches + "/0026-src-boot-efi-efi-string.c-define-wchar_t-from-__WCHA.patch")
+      (musl-patches + "/0025-include-sys-file.h-for-LOCK_EX.patch")
+      (musl-patches + "/0026-test-test-sizeof-Include-sys-timex.h-for-struct-time.patch")
+      (musl-patches + "/0027-include-missing-sys-file.h-for-LOCK_EX.patch")
+      (musl-patches + "/0028-sd-event-Make-malloc_trim-conditional-on-glibc.patch")
+      (musl-patches + "/0029-shared-Do-not-use-malloc_info-on-musl.patch")
     ]
   );
```

Reply:
If you can pick it up from there in a follow-up, that'd be awesome!

Reply:
It seems BTW we are lacking:

> -      (musl-patches + "/0026-src-boot-efi-efi-string.c-define-wchar_t-from-__WCHA.patch")

as the error is related to wchar redefinition stuff.